Ficor update

cacraig opened the pull request:
- Backed up current data in new format.
- Adjusted Pass list reader to handle newly formatted data.
- Added new pass list, and backup list to nationalparks/data

Commit:
wip

Commit:
Backed up passes list, and added new list

Commit:
Removed print + whitespace

Commit:
Added missing header row to backup date

cacraig commented:
@khandelwal Any input, or are we good to go on this ?

khandelwal commented:
Let me check

khandelwal commented on ekip/nationalparks/management/commands/passes.py:
```diff
@@ -105,14 +108,16 @@ def read_pass_list(filename):
             passcsv, fieldnames=field_names, delimiter=',') 
 
+        header = True
         for l in passreader:
```
Calling next() on the iterator here, before the for loop will skip the first row of the CSV instead of doing the header=True, header=False thing.

khandelwal commented on ekip/nationalparks/management/commands/passes.py:
```diff
@@ -73,17 +73,20 @@ def process_site(site, next_version):
         fs.website = site['website']
         fs.annual_pass = annual
-        fs.senior_pass = senior
+        fs.senior_pass = annual
```
This is confusing from a self-documenting code standpoint. Why is annual set to senior_pass? I'd either rename things, or add some comments to make this clear.

khandelwal commented on ekip/nationalparks/management/commands/passes.py:
```diff
@@ -73,17 +73,20 @@ def process_site(site, next_version):
-    random_site = FederalSite.objects.filter(active_participant=True)[0]
-    return random_site.version + 1
+    
+    random_site_version = 1
```
Should the default here be 1 or 0?

cacraig replied:
I left it at 1, but removed the increment on the return val.

khandelwal replied:
Ah, no. You want the increment.

khandelwal replied:
So you'll have to change this.

cacraig replied:
I mispoke, I mean it's still there. Just It's basically 1 OR last_version + 1. I guess I meant to say I moved it from the return value to conditional.

khandelwal replied:
Ah, I see. My bad.

khandelwal commented:
Code has changed, but there are no corresponding tests. It would be nice to add some tests in.

Commit:
Added unit tests, and clarification comments

cacraig commented:
@khandelwal I added a few tests, and modified according to your suggestions. I left the FederalSite.senior field in there to maintain backwards compatibility. I'd like to remove it entirely, but i'd like to make sure it's not going to break anything or if it will be needed from here on out.

khandelwal commented:
Also, I'm not seeing any unit tests. Did they not make the commit?

Commit:
Added tests

cacraig commented:
Nope! They are there now @khandelwal .
